packages/chat/src/components/Header/index.tsx

```diff
@@ -36,20 +36,15 @@ export interface HeaderProps {
    * A list of actions that will appear as icon buttons.
    */
   actions?: HeaderActionProps[];
-
-  /**
-   * The primary color of the header.
-   */
-  rounded?: boolean;
 }
 
 /**
  * Header for the chat widget with image, title and controls.
  *
  * @see {@link https://voiceflow.github.io/react-chat/?path=/docs/components-chat-header--simple}
  */
-export const Header: React.FC<HeaderProps> = ({ title, image, actions = [], rounded }) => (
-  <div className={clsx(ClassName.HEADER, headerContainer({ rounded }))}>
+export const Header: React.FC<HeaderProps> = ({ title, image, actions = [] }) => (
+  <div className={clsx(ClassName.HEADER, headerContainer)}>
```

Comment:
removed `rounded` cause this isn't necessary - the outer container has `overflow: hidden` so it's round corners are dependant on that.